### DISCUSSION TITLE
Is there a way to pull files from one board to another like in Kanban

### DISCUSSION THREAD

**Discussion by EliasSchlie** [marcusolsson/obsidian-projects#353] [Help]
The same as in caleandar view, where you can pull files from one date to another and it get's changed in the note. 

Would be handy to use like Kanban. Is this possible?

**Comment by marcusolsson** [comment]
If you drag and drop a card from one column to another, it updates the notes. Is that what you mean?

**Reply by SimpleMedia1** [reply]
I'm also not able to drag and drop in the Boards view

**Reply by marcusolsson** [reply]
Are you using Dataview maybe? Dataview projects are readonly at the moment.

**Comment by JordanChoe** [comment]
This has something to do with the priority drop down I believe.  I haven't figured out where the documentation is to control what field options for priority drop down.  I'm not using dataview projects. I've abandoned it for the time being because when creating board views it is updating the yaml to no status for every file in no status column for the field I am grouping by after a priority field is selected (a weird one off random yaml field ironically that was used in 1 of 570 or so files in the project table and database).  This is not desired for my use case.

**Reply by marcusolsson** [reply]
> it is updating the yaml to no status for every file in no status column for the field 

I'm not sure I understand what you mean here. Would you be willing to open an issue with some details on how to reproduce this?

**Comment by mdmeadows** [comment]
I have the same issue in the Board view, if I set the "Priority" dropdown to "None" (not really needed in some of my projects). If Priority is set to None, clicking-and-dragging a card moves the whole column. As soon as I set "Priority" to anything other than "None", dragging individual cards from one column to another works as expected again.

**Reply by marcusolsson** [reply]
I've written #451 under FAQ that explains the reasoning for this.

**Comment by TomWol** [comment]
I have the same issue, without setting a priority I am unable to drag cards, only columns. Not using Dataview in anything, even in the "Demo project" included with the installation, if I change priority to "None", I lose the ability to drag cards.

**Reply by marcusolsson** [reply]
I've written #451 under FAQ that explains the reasoning for this.
